cluster-manifests/.flux.yaml

```diff
@@ -0,0 +1,5 @@
+version: 1
+patchUpdated:
+  generators:
+    - command: kustomize build .
+  patchFile: flux-patch.yaml
```

Comment:
Is this still necessary now? The files appear to be empty.

Reply:
yeah, there were empty. Me being lazy this time but I won't allow myself be that way in the case 😸

done | addressed from [5f7f2](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/5f7f2099c0d99a0c3d179f7db1170fea2b831cf6)

Reply:
Ha, "Lazy" is one of my seven middle names, of which I'm too lazy to list here :P

Reply:
I've reinstated these lazy files. Otherwise, Flux is going to log some errors since the `patchFile` seems to be required.

reverted | Addressed from [60079](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/60079f43c04933a6a337c6ebfeeb893996f45822)